distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -166,10 +166,12 @@ grant {
     permission java.io.FilePermission "${ddf.home}${/}solr", "read, write";
     permission java.io.FilePermission "${ddf.home}${/}system", "read, write";
     permission java.io.FilePermission "${ddf.home}${/}workspace", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}content", "read, write";
```

Comment:
:question: This works for the default content store, but what happens when an admin wants to change it?  Is there documentation regarding changing the security manager policy?  For instance, in a Highly Available setup, the content store is shared by both nodes.

Reply:
This will be documented by @ricklarsen

Reply:
❗️ Not sure we should grant everybody these permissions. Can we limit them to the proper codebase?